### PR TITLE
CPS-0005 | Plutus Script Usability

### PR DESCRIPTION
**Pull request by michaelpj**
This CPS pulls together motivation from a number of CIPs, with the aim of describing a nubmer of issues in once place, rather than having them repeated amongst the many CIPs addressing various aspects of the problem.

---

[rendered proposal in branch](https://github.com/michaelpj/CIPs/blob/mpj/script-interaction-cps/CPS-%3F%3F%3F%3F/README.md)

### CODE REVIEW

File: CPS-????/README.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+### Native script payments
+
+Charlie wants to make a payment to a native script address. 
+This doesn’t require a datum, so they can do it simply based on the address, but their wallet gives them a scary warning because it doesn’t know that the address isn’t a Plutus script address that requires a datum.
+
+### Smart contract wallet
+
+Eve has no idea about key security, but cares about custodianship and hence prefers to use a wallet which allows her to recover funds using a social recovery key sharing scheme. 
+This system uses a Plutus script to be the “owner” of the funds. 
+When Eve wants to receive funds, her friends should not need to know what kind of wallet she is using - she should be able to provide them with a simple way to pay that is not meaningfully harder or different to paying into a normal wallet.
```
Comment (ch1bo) [inline CPS-????/README.md:122]: Thanks for including my use case :)

**Review by rphair — APPROVED** [review]
This looks like a perfectly phrased CPS to me, so approving on that basis.  Before merging maybe some third parties can double-check that all related CIPs and use cases are included.  For convenience of both official & community review here are the threads for the pending CIPs currently in the preamble:

- https://github.com/cardano-foundation/CIPs/pull/309 @SebastienGllmt
- https://github.com/cardano-foundation/CIPs/pull/258 @KtorZ
- https://github.com/cardano-foundation/CIPs/pull/321 @zygomeb
- https://github.com/cardano-foundation/CIPs/pull/440 @fallen-icarus

and tagging frequent contributors to the above: @JaredCorduan @nielstron @L-as

**Comment by JaredCorduan** [comment]
thanks @rphair , I also think that this CPS is great and I've nothing to add.

**Comment by L-as** [comment]
I think it's a bit orthogonal. This CPS is about the users' perspective, whereas that is about the developers' perspective.

**Comment by michaelpj** [comment]
Any comments from @SebastienGllmt ?

**Comment by NetWalker108** [comment]
Well written @michaelpj

**Review by KtorZ — APPROVED** [review]

**Pull request merged**